src/Engine/Utils.java

```diff
@@ -719,7 +719,7 @@ public static void enqueueDeathEvent(Unit unit, GameEventQueue eventQueue)
   /**
    * @return Whether the owner of this property will lose if the property is lost
    */
-  public static boolean willLoseFromLossOf(MapMaster map, MapLocation capLoc)
+  public static boolean willLoseFromLossOf(GameMap map, MapLocation capLoc)
```

Comment:
I thought you didn't want to change the signature of this function, since we want to have a `MapMaster` if we are making win/loss decisions.

Reply:
I didn't.
However, I realized that there are valid usecases for checking that sort of question without immediately generating events - for example, this is probably more reliable than "is this a lab or HQ?" for AIs to make decisions on.

Reply:
But there are places where this is invoked with just a `GameMap` - doesn't this mean that if a Kaiju crushes a lab, and doesn't know about any other labs, then in `KaijuActions::271` it'll decide there must not be any other labs and enqueue the defeat event?

Reply:
Yes and no.
Yes, that code will be executed. The event will be generated and queued.
The event will *not* be executed, because `getEvents()` requires a `MapMaster`, and the event queue generated by this function is only used by `getEvents()`.

On reflection, I don't think it's the job of a util function to make sure you do the right thing with its output - it should only be concerned with giving you the right answer to the question you asked.

Reply:
OK, I may buy that. Or maybe I'm just tired of seeing the word "Kaiju". Either way, go ahead and merge this stuff if you're happy with it.